performance.md

```diff
@@ -29,10 +29,10 @@ Global optimizations are used to speed up the overall messages processing improv
 
 Rspamd uses various methods to speed up each individual message processing stage. This is achieved by applying local optimizations techniques:
 
-* **<abbr title="Abstract Syntax Tree">AST</abbr> optimizations** are used to exclude unnecessary checks from rules. You can watch the following [slides](http://www.slideshare.net/VsevolodStakhov/astrspamd) to get more details about this method.
+* **<abbr title="Abstract Syntax Tree">AST</abbr> optimizations** are used to exclude unnecessary checks from rules. You can watch the following [slides](https://www.slideshare.net/VsevolodStakhov/astrspamd) to get more details about this method.
```

Comment:
This page is unavailable: https://www.slideshare.net/VsevolodStakhov/astrspamd

Reply:
Same as above, it's available